src/lib/pipeline/m2/index.js

```diff
@@ -8,39 +9,75 @@ class M2 extends THREE.Group {
 
   static cache = {};
 
-  constructor(path, data, skinData) {
+  constructor(path, data, skinData, instanceOpts) {
     super();
 
+    const instance = instanceOpts || null;
+
+    this.name = path.split('\\').slice(-1).pop();
```

Comment:
In my console when I'm outputting an M2 to inspect. ;)

It's a valid property of ```THREE.Object3D```:
http://threejs.org/docs/#Reference/Core/Object3D.name

I figure we'll eventually want to create some basic scene tools to let us hunt for different things within a scene and inspect properties on them. It'll go a long way toward understanding why something is the way it is, and might pave the way for our very own version of ```WowEdit``` some day.